paderbox/io/audioread.py

```diff
@@ -166,22 +167,35 @@ def load_audio(
         raise ValueError(unit)
 
     try:
-        with soundfile.SoundFile(
-                path,
-                'r',
-        ) as f:
-            if dtype is None:
-                from paderbox.utils.mapping import Dispatcher
-                mapping = Dispatcher({
-                    'PCM_16': np.int16,
-                    'FLOAT': np.float32,
-                    'DOUBLE': np.float64,
-                })
-                dtype = mapping[f.subtype]
-
-            frames = f._prepare_read(start=start, stop=stop, frames=frames)
-            data = f.read(frames=frames, dtype=dtype, fill_value=fill_value)
-        signal, sample_rate = data, f.samplerate
+        if path[-3:] == 'm4a':
+            with ar.audio_open(
+                    path
+            ) as f:
```

Comment:
Can you add an assert that start and stop aren't supported? And add a comment, that audioread does not support them.